packages/kotti-ui/source/kotti-table/types.ts

```diff
@@ -97,6 +97,14 @@ export module KottiTable {
 			) => VNode
 		}
 
+		export type PropsInternal = Omit<Props, 'renderCell' | 'renderHeader'> & {
+			renderCell: (h: CreateElement, data: Context) => VNodeChildren
```

Comment:
@Isokaeder Why not VNode here ?

In Yoda, we use some renderCell functions which render VNode type.